packages/cht/package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "@openfn/language-cht",
+  "label": "Community Health Toolkit",
```

Comment:
Should this just say `CHT` ?. On their [website](https://communityhealthtoolkit.org/) it says `Community Health Toolkit (CHT)`

Reply:
Let's do that, let's do `Community Health Toolkit (CHT)`